### PR TITLE
small fixes to the tutorials

### PR DESCRIPTION
**Pull request by tomerfqm**

### CODE REVIEW

File: Tutorials/frame-and-phase-intro/reset_phase_demo.py [hunk 1]
```diff
@@ -15,7 +14,7 @@
 from configuration import *
 from scipy.optimize import leastsq
 
-qmm = QuantumMachinesManager(port=9510)
+qmm = QuantumMachinesManager()
```
Comment (yomach) [inline Tutorials/frame-and-phase-intro/reset_phase_demo.py:17]: I only wonder about this...
WDYT?

Comment (yomach) [reply Tutorials/frame-and-phase-intro/reset_phase_demo.py:17]: Not about removing the port (this is needed!), but about not having IP...

Comment (tomerfqm) [reply Tutorials/frame-and-phase-intro/reset_phase_demo.py:17]: What do you want to write there? `qop_ip`

Comment (tomerfqm) [reply Tutorials/frame-and-phase-intro/reset_phase_demo.py:17]: I removed it because it was a local thing on my end and I wanted it to be general..

Comment (TheoLaudatQM) [reply Tutorials/frame-and-phase-intro/reset_phase_demo.py:17]: I am a bit confused, so now we don't need to specify the IP at all ?

Comment (yomach) [reply Tutorials/frame-and-phase-intro/reset_phase_demo.py:17]: We need to specify an IP. I'm just saying that we should have a default value inside the `QuantumMachinesManager()` such that the syntax will be correct

**Review by TheoLaudatQM — APPROVED** [review]

**Pull request merged**